::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2018 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Wed, 22 Aug 2018 01:35:06 +0200
Subject: [PATCH] OPNFV package installation, Ubuntu user

Install OpenSSH server (and client), so other OPNFV projects can
easily connect to the Salt master node.

While at it, create 'ubuntu' user so other OPNFV projects don't
have to switch to 'root' login.

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 DockerMake.yml | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/DockerMake.yml b/DockerMake.yml
index 2c75586..1d34932 100644
--- a/DockerMake.yml
+++ b/DockerMake.yml
@@ -108,6 +108,19 @@ salt-formulas:
       && bash -c 'source /srv/salt/formula-fetch.sh && setupPyEnv && fetchAll' \
       && eval ${LAYER_CLEANUP}

+opnfv:
+  build: |
+    RUN echo "Layer with OPNFV packages" \
+      && eval ${LAYER_PKGUPDT} \
+      && ${LAYER_INSTALL} \
+           openssh-server \
+           gawk \
+           kmod \
+      && useradd -m ubuntu \
+      && echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' > /etc/sudoers.d/ubuntu \
+      && eval ${LAYER_CLEANUP}
+
+
 wheel:
   requires:
     - base
